### ISSUE TITLE
Navigation

### ISSUE THREAD

**Issue by michalCic** [lubsar/AndroidUIs#1]
Nice UIs, but no way of navigating in between them... :D

**Comment by lubsar** [comment]
Fair point, but I abandoned this project before it became something, so no promises on fixing this

**Comment by michalCic** [comment]
Such a shame, it looked promising
